packages/forma-36-codemod/transforms/v5/icons.js

```diff
@@ -0,0 +1,147 @@
+const {
+  getComponentLocalName,
+  changeComponentName,
+  changeImport,
+} = require('../../utils');
+const { shouldSkipUpdateImport, getImport } = require('../../utils/config');
+
+// V4 icon name : V5 icon name
+const iconsMap = {
+  ArrowBackward: 'ArrowLeft',
+  ArrowDown: 'CaretDownFill',
+  ArrowDownward: 'ArrowDown',
+  ArrowForward: 'ArrowRight',
+  ArrowUp: 'CaretUpFill',
+  ArrowUpward: 'ArrowUp',
+  Audio: 'FileAudio',
+  Calendar: 'CalendarBlank',
+  ChatBubble: 'ChatFill',
```

Comment:
We might need to double-check that list when we are ready, for instance, this v5 icon is not part of https://github.com/contentful/forma-36/pull/2647 yet (we use `Chat` 🐱 ).

Reply:
I imagine we will be updating this mapping a few times no matter what. We need to press design for a final list though.